doc/kernel/other/interrupts.rst

```diff
@@ -106,8 +109,12 @@ as A, B, C, and D:
    D -> 0x00030609
 
 .. note::
-   The bit positions for LEVEL 2 and onward are offset by 1, as 0 means that
-   interrupt number is not present for that level.
+    The bit positions for LEVEL 2 and onward are offset by 1, as 0
+    means that interrupt number is not present for that level. ie, the
```

Comment:
Change ``ie, the`` and the rest of the paragraph to:

    For our example, the LEVEL 3 controller has device D
    on line 2, connected to the LEVEL 2 controller's line 5, that
    is connected to the LEVEL 1 controller's line 9 (2 -> 5 -> 9).
    Because of the encoding offset for LEVEL 2 and onward,
    device D is given the number 0x00030609.

Reply:
fixed.